infrastructure/docker/celo/start.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+mkdir -p /data/keystore
+cp init/keystore/* /data/keystore
+cp init/password.txt /data/password.txt
+[ ! -d "/data/celo" ] && geth --nousb --datadir=/data init init/genesis.json
+
+ geth --port 3000 --networkid 1101 --identity "somerandomidentity" \
```

Comment:
Some day we may want to make some properties (e.g. ports and network id) configurable (with defaults) in case we want to run it along with an Ethereum client (geth or ganache) or run it outside of the local dev machine (e.g. keep test envs).
Taking into account the first case, would it be worth doing now?

Reply:
In `truffle-config.js` we have just one local network, for Celo and Ethereum running on port 8545, so I'd rather prefer to keep it as-is and aligned with truffle config. I am afraid making it parametrizable does not help that much now but can cause some confusion during the environment setup. https://github.com/keep-network/keep-core/blob/61f9bb7012bf583b09afb63b0655b230cdadcf25/solidity/truffle-config.js#L8-L12